scripts/debug_tools/prepare_compiler_info.py

```diff
@@ -20,17 +20,19 @@ def prepare(compiler_info_file, sources_root):
     sources_root_abs = os.path.abspath(sources_root)
     new_json_data = dict()
     for compiler in json_data:
-        lines = json_data[compiler]['includes']
-        changed_lines = []
-        for line in lines:
-            changed_lines.append(
-                lib.change_paths(line,
-                                 lib.IncludePathModifier(sources_root_abs)))
-        new_json_data[compiler] = {
-            'includes': changed_lines,
-            'target': json_data[compiler]['target'],
-            'default_standard': json_data[compiler]['default_standard']
-        }
+        new_json_data[compiler] = dict()
+        for language in json_data[compiler]:
+            lines = json_data[compiler][language]['compiler_includes']
```

Comment:
Ah, I remember when @gyorb added the `language` stuff... (that is the time when it broke, but we forgot then)

Reply:
Yes, probably the update for this script was left out at that time.